applications/ecallmgr/src/ecallmgr_fs_command.erl

```diff
@@ -26,8 +26,10 @@
 %% set channel and call variables in FreeSWITCH
 %% @end
 %%--------------------------------------------------------------------
--spec set(atom(), ne_binary(), kz_proplist()) -> ecallmgr_util:send_cmd_ret().
+-spec set(atom(), api_ne_binary(), kz_proplist()) -> ecallmgr_util:send_cmd_ret().
 set(_, _, []) -> 'ok';
+set(_Node, 'undefined', _Props) ->
+    lager:debug("no UUID for setting on node ~s: ~p", [_Node, _Props]);
```

Comment:
@jamesaimonetti can we raise the level to `warning` ? if this happens there are problems elsewhere that need to be verified